tool/RELEASE_INSTRUCTIONS.md

```diff
@@ -168,3 +173,77 @@ have unreleased changes, publish these packages to pub.
 [README.md](https://github.com/flutter/devtools/blob/master/packages/devtools_app/release_notes/README.md)
 to add DevTools release notes to Flutter website and test them in DevTools.
 2. Once release notes are submitted to the Flutter website, send an announcement to g/flutter-internal-announce with a link to the new release notes.
+
+## Cherry-pick releases
+
+### Prepare the release in the `flutter/devtools` repo
+
+Find the [DevTools tag](https://github.com/flutter/devtools/tags) that you want to perform the cherry-pick release on top of.
+Then checkout that tag locally. For this example, we'll use `v2.29.0` as the base branch and `2.29.1` as the cherry-pick branch.
+
+```
+git checkout v2.29.0
+```
+
+Once checked out, create a new branch for your cherry picks. 
+
+```
+git checkout -b 2.29.1
+```
+
+Cherry pick the commit(s) you want in this cherry-pick release, and bump the DevTools version number:
+```
+git cherry-pick <commit>
+devtools_tool update-version auto -t patch
+```
+
+Commit your changes and push to the `upstream` remote.
+
+```
+git add .
+git commit -m "Prepare cherry-pick release - DevTools 2.29.1"
+git push upstream 2.29.1
+```
+
+Once you are completely satisfied with your changes, create a tag for this cherry-pick release:
+```
+tool/tag_version.sh
+```
+
+To move on to the next step, you will need to take note of two values:
+1) The name of the DevTools tag you just created (e.g. `v2.29.1`)
+2) The commit hash that is at the tip of this tag (see https://github.com/flutter/devtools/tags).
+
+### Manually run the DevTools Builder
+
+Follow the instructions at [go/dart-engprod/devtools.md#cherry-picks](go/dart-engprod/devtools.md#cherry-picks)
```

Comment:
It's probably worth mentioning that this step can only be completed by Googlers on the Dash team, especially since that go link won't work externally.

Reply:
There also seems to be some other go links in this document. Maybe leave a disclaimer at the top of the file stating that these instructions will only work for Googlers?